en/components/chip.md

```diff
@@ -165,13 +165,22 @@ The chip can be focused using the `Tab` key or by clicking on them. Chips can be
   - <kbd>LEFT</kbd> - Focuses the chip on the left
   - <kbd>RIGHT</kbd> - Focuses the chip on the right
   - <kbd>SPACE</kbd> - Toggles chip selection if it is selectable
-  - <kbd>DELETE</kbd> - Fires the `onRemove` output so the chip deletion can be handled manually
+  - <kbd>DELETE</kbd> - Fires the [`onRemove`]({environment:angularApiUrl}/classes/igxchipcomponent.html#onremove) output so the chip deletion can be handled manually
   - <kbd>SHIFT</kbd> + <kbd>LEFT</kbd> - Moves the focused chip one position to the left
   - <kbd>SHIFT</kbd> + <kbd>RIGHT</kbd> - Moves the focused chip one position to the right
 
 - Keyboard controls when the remove button is focused:
 
-  - <kbd>SPACE</kbd> or <kbd>ENTER</kbd> Fires the `onRemove` output so the chip deletion can be handled manually
+  - <kbd>SPACE</kbd> or <kbd>ENTER</kbd> Fires the [`onRemove`]({environment:angularApiUrl}/classes/igxchipcomponent.html#onremove) output so the chip deletion can be handled manually
+
+### API
+
+* [IgxChipComponent]({environment:angularApiUrl}/classes/igxchipcomponent.html)
```

Comment:
Provide after this line a Sassdoc reference to the igx-chip-theme as follows:
[IgxChipComponent Styles] ({environment:sassApiUrl}/index.html#function-igx-chip-theme)